wp-content/themes/core/parts/header.html

```diff
@@ -1 +1,10 @@
-Header
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+	<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|40"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+	<div class="wp-block-group alignwide" style="padding-bottom:var(--wp--preset--spacing--40)">
+		<!-- wp:site-title {"level":0} /-->
+		<!-- wp:navigation {"layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right"}} /-->
```

Comment:
Added this purely to see how my button and link styles affected the site title and nav blocks. I can remove it, if needed.